akka-actor/src/main/scala/akka/pattern/RetrySupport.scala

```diff
@@ -30,8 +30,7 @@ trait RetrySupport {
    * private val sendReceiveRetry: HttpRequest => Future[HttpResponse] = (req: HttpRequest) => retry[HttpResponse](
    *   attempt = () => sendAndReceive(req),
    *   attempts = 10,
-   *   delay = 2 seconds,
-   *   scheduler = context.system.scheduler
+   *   delay = 2 seconds
```

Comment:
I think this is better per the latest convention
```suggestion
   *   delay = 2.seconds
```